packages/flutter/test/cupertino/dialog_test.dart

```diff
@@ -344,15 +363,21 @@ void main() {
     // regular font. However, when using the test font, "Cancel" becomes 2 lines which
     // is why the height we're verifying for "Cancel" is larger than "OK".
 
-    if (!kIsWeb || isCanvasKit) { // https://github.com/flutter/flutter/issues/99933
-      expect(tester.getSize(find.text('The Title')), equals(const Size(270.0, 132.0)));
+    if (!kIsWeb || isCanvasKit) {
+      // https://github.com/flutter/flutter/issues/99933
```

Comment:
This comment pointing to an issue should stay attached to the end of the first line.